### PR TITLE
Make economic tariffs used for a given school more transparent on es

### PR DESCRIPTION
**Pull request by ldodds**
This PR revises the current page that displays the manually entered tariffs for a school to do several things:

* Improve the layout
* Add a tab to show the tariffs automatically loaded from smart mters
* Add a tab to show the default system wide tariffs

This is to make all the tariff information much more accessible to users. Previously the detail has been split across the tariff and costs pages.

This is still a work in progress:

* [x] Revise how tariff prices are displayed for smart meter tariffs
* [x] Ensure new text is localised
* [x] On first tab, add a reference to smart meter tariffs if we have them
* [x] Implement the "default tariffs" tab
* [x] add link to page

### CODE REVIEW

**Comment by ldodds** [comment]
This has been merged to aws-eb-test, so we can get some feedback.

File: app/helpers/tariffs_helper.rb [hunk 1]
```diff
@@ -1,4 +1,54 @@
...
+
+  #find all of the economic tariffs for a specific group, collecting them
+  #via their scope
+  def economic_tariffs_for_school(school)
+    tariffs = {}
+    tariffs[:school] = convert_meter_attributes_to_hash(school.pseudo_meter_attributes)
+    tariffs[:school_group] = convert_meter_attributes_to_hash(school.school_group_pseudo_meter_attributes)
+    tariffs[:global] = convert_meter_attributes_to_hash(school.global_pseudo_meter_attributes)
+    tariffs
+  end
```
Comment (iant) [inline app/helpers/tariffs_helper.rb:14]: possible simplification
```
  def economic_tariffs_for_school(school)
    {
      school: convert_meter_attributes_to_hash(school.pseudo_meter_attributes),
      school_group: convert_meter_attributes_to_hash(school.school_group_pseudo_meter_attributes),
      global: convert_meter_attributes_to_hash(school.global_pseudo_meter_attributes)
    }
  end
```

File: app/views/schools/user_tariffs/_smart_meter_tariffs_tab.html.erb [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+            <%= t('schools.user_tariffs.index.electricity.header') %>
+          </h4>
+          <%= render 'smart_meter_tariffs_summary_table', meters: @electricity_meters.dcc, school: @school %>
+        <% end %>
+      </div>
+    </div>
+
+    <div class="row">
+      <div class="col">
+        <% if @gas_meters.dcc.any? %>
```
Comment (iant) [inline app/views/schools/user_tariffs/_smart_meter_tariffs_tab.html.erb:22]: same as above - can this wrap the `<div class="row">...</div>` bit?

**Review by iant — APPROVED** [review]
hi @ldodds  this looks ok to me.  couple of comments on possible improvements but if this is going to be re-visted in the next few weeks i think all ok to go in as-is.

**Pull request merged**